contracts/axelar-gateway/src/auth.rs

```diff
@@ -1,6 +1,7 @@
 use crate::error::ContractError;
```

Comment:
PR title should be `refactor(axelar-gateway): use typed events` to adhere to this convention
https://www.conventionalcommits.org/en/v1.0.0/#summary